### PR TITLE
Init.

### PR DESCRIPTION
**Pull request by vivekimsit**
Changelog:

Added a method `truncate` which takes optional `limit` which defaults to `40` and an optional `separator` which defaults to `...`

### CODE REVIEW

File: objects/github_tickets.rb [hunk 1]
```diff
@@ -99,6 +99,14 @@ def users
     end
   end
 
+  def truncate(text, limit = 40, separator = '...')
+    return text if limit >= text.length
+
```
Comment (yegor256) [inline objects/github_tickets.rb]: @vivekimsit remove this empty line, see http://www.yegor256.com/2014/11/03/empty-line-code-smell.html

**Comment by yegor256** [comment]
@vivekimsit looks good, but there is one comment from me, see above

**Comment by vivekimsit** [comment]
I think there is some issue in travis.

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by yegor256** [comment]
@rultor stop

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**